src/Proto3/Wire/Decode.hs

```diff
@@ -501,14 +519,13 @@ one parser def = Parser (fmap (fromMaybe def) . traverse (runParser parser) . pa
 --
 -- For example, to parse a packed collection of @uint32@ values:
 --
--- > repeated uint32 :: Parser RawField (Seq Word32)
+-- > repeated uint32 :: Parser RawField ([Word32])
 --
 -- or to parse a collection of embedded messages:
 --
--- > repeated . embedded' :: Parser RawMessage a -> Parser RawField (Seq a)
-repeated :: Parser RawPrimitive a -> Parser RawField (Seq a)
-repeated parser = Parser $
-  fmap fromList . mapM (runParser parser) . toList
+-- > repeated . embedded' :: Parser RawMessage a -> Parser RawField ([a])
+repeated :: Parser RawPrimitive a -> Parser RawField [a]
+repeated parser = Parser $ fmap reverse . mapM (runParser parser)
```

Comment:
I'm curious: do you have benchmark data showing the effect of changing from `Seq` to `[]`?

Reply:
No. This is just a "best practices" and "first principles" thing based on auditing the code and seeing usage patterns.

Reply:
Well, `reverse` has a cost too, so I don't think it's obvious.  At some point we need performance numbers, even crude ones.